qt_files/comp_app.ui

```diff
@@ -207,6 +207,67 @@
     </layout>
    </widget>
   </widget>
+  <widget class="QDockWidget" name="dockWidget_4">
```

Comment:
If you cause the dockedwidget to be resized, either by resizing the entire window or pulling out the docked widget and resizing it, the slider gets cut out if you go smaller and leaves a lot of empty space if you go bigger.

Would be nice to make it like the buttons where it stretches to fill the empty space. And also like the labels so that there's a limit on how small you can go. I don't remember all the details of how to do this, but I'm sure you can reverse engineer those. I think you might have to add a layout for that to work or something.